docs/reference/analysis/analyzers/stop-analyzer.asciidoc

```diff
@@ -1,9 +1,15 @@
 [[analysis-stop-analyzer]]
 === Stop Analyzer
 
-The `stop` analyzer is the same as the <<analysis-simple-analyzer,`simple` analyzer>>
-but adds support for removing stop words.  It defaults to using the
-`_english_` stop words.
+The `stop` analyzer is the same as the <<analysis-simple-analyzer,
+`simple` analyzer>> but adds support for removing stop words.  It defaults to
+using `_english_` stop words.
+
+Stop words are common words set not to be indexed. By removing a list of common
+words, you can reduce the size of an index and increase performance, which is
+important when disk space and memory are critical factors.
+
+
 
 [float]
```

Comment:
Probably want to add an anchor here like `[[stop-analyzer-example-output]]`

Reply:
> Stop words are common words set not to be indexed. By removing a list of common
words, you can reduce the size of an index and increase performance, which is
important when disk space and memory are critical factors.

This section is a bit misleading since we added optimizations in Lucene 8 to speed up queries that contain stop words. Removing stop words is also challenging for other token filters like `synonym_graph` and `synonym` so now that we are able to run queries with stop words efficiently I am not sure we should add such advice in the docs. We could maybe just mention the disk space savings and add a note regarding the faster search if the total hits that match the query is not requested (making the removal of stop words not necessary) ?